_posts/2019-03-19-Kickstarter-Trajectory-2019-Light.md

```diff
@@ -0,0 +1,296 @@
+
+For lunch we went to a place called Meals On Wheels were we were introduced to a whole other world of sandwiches.
+Once you’ve been there you will know what we mean by that, don’t go too often though.
+
+## Kubernetes
+<img class="image left medium" alt="Kubernetes Logo" src="/img/2019-Kickstarter-Trajectory-2019-light/kubernetes-logo.png" />
+
+[Kubernetes](https://kubernetes.io/) is an open source container orchestration framework which was first introduced to us by [Tom Verelst](/author/tom-verelst) during the kickstarter traject.
+It is made by Google and is now maintained by the Cloud Native Computing Foundation.
+First they introduced us to all the features that Kubernetes possesses (service discovery, horizontal scaling, load balancing, ...). 
+
+Soon we learned how to deploy Docker containers in the cloud by using Kubernetes, and afterwards we had an hands-on exercise where we could deploy a full-stack application to a Kubernetes cluster using [Minikube](https://github.com/kubernetes/minikube).
+It's wonderful how you can deploy a full-stack application through Kubernetes with just one configuration file needed.
+Of course, it takes some time to get used to it, but once you get the hang of it, you can do outstanding stuff with this platform!
+
+## Cloud Providers & Platforms
+<img class="image left medium" alt="AWS Logo" src="/img/2019-Kickstarter-Trajectory-2019-light/aws-logo.png">
+To get a bigger picture of all the cloud providers and platforms that are out there conquering the IT world, 
+we had a dedicated session about this topic given by [Bas Moorkens](/author/bas-moorkens) and [Dieter Hubau](/author/dieter-hubau).
+Bas was focusing on Amazon Web Services and all its features that it has to offer.
+
+We quickly learned that AWS was very advanced and had lots of components to facilitate the life of a developer.
+It was a very interesting session and made me realise that AWS was a big part of the development scene.
+We are eager to use it and learn more of what is has to offer. 
+
+<img class="image right medium" alt="OpenShift Logo" src="/img/2019-Kickstarter-Trajectory-2019-light/openshift-logo.png">
+
+As for cloud platforms, we got a very detailed explanation of how OpenShift (made by RedHat) works and what its features and options are.
+We also got a high-level explanation as to how an application in the cloud works and what the best practices are to achieve deploying your application in the cloud.
+
+Overall, it was a very interesting session for cloud-enthusiasts and we definitely want to learn more about it!
+
+## TypeScript
+After the session HTLML5, CSS3, JavaScript, [Dimitri De Kerf](/author/dimitri-de-kerf) learned us the benefits of TypeScript. 
+He told us some benefits of using TypeScript instead of JavaScript. This means that TypeScript is a wrapper around JavaScript, 
```

Comment:
```suggestion
He told us some benefits of using TypeScript instead of JavaScript.
TypeScript is a wrapper around JavaScript, 
```

Reply:
Also a major advantage is that the code gets compiled to JavaScript meaning that you'll get compiler errors whereas in JavaScript you'll have to run into the issue at runtime.
Type safety is also an advantage as it allows your IDE to better assist you with code completion.